src/com/sun/jna/Structure.java

```diff
@@ -865,20 +871,79 @@ protected void writeField(StructField structField) {
         }
     }
 
-    /** Return this Structure's field names in their proper order.  For
-     * example,
+    /** Used to decllare fields order as metadata instead of method.
+     * example:
      * <pre><code>
-     * protected List getFieldOrder() {
-     *     return Arrays.asList(new String[] { ... });
+     * // New
+     * <!-- -->{@literal @}FieldOrder({ "n", "s" })
```

Comment:
Is there a reason for these empty html comments?

Reply:
**fixed**

Yes... in vscode if a javadoc line starts with a '@' it implodes all the code lines into one line, You are right I should remove that and hope that vscode fix this problem.
https://github.com/Microsoft/vscode/issues/48898